src/odemis/gui/cont/tabs.py

```diff
@@ -2034,8 +2034,68 @@ def display_new_data(self, filename, data, extend=False):
                                 self.tab_data_model.ar_cal.value)
                 self.tab_data_model.ar_cal.value = u""  # remove the calibration
 
-        # TODO: if all the views are either empty or contain the same streams,
+        # if all the views are either empty or contain the same streams,
         # display in full screen by default (with the first view which has streams)
+        # Show the 1 x 1 view if:
+        # - the streams are all the same in every view 
+        # - views are empty with no projection
+        
+        def compare_projection_lists(list1, list2):
+            """ Returns true if two projection lists are the same, 
+            i.e. they are projecting the same streams, even if the objects are different """
```

Comment:
Maybe I dumb question,but if you only check if two streams are a projection from the same stream would simply checking the stream name not be sufficient?

Reply:
The projections could be different, so in fact it wouldn't give the desired effect. For instance, one spectrum stream could be projected in several different ways (line spectrum, point spectrum, etc.)